packages/cockpit/src/recipient/detail/format/index.js

```diff
@@ -33,9 +33,18 @@ function formatAntecipationAndTransferConfiguration (data) {
     documentType: data.bank_account.document_type,
   }
 
+  let cpfPhone
+  let cnpjPhone
   const register = data.register_information
   const phone = pathOr('', ['phone_numbers', 0, 'number'], register)
```

Comment:
Acho que poderia coletar daqui o `ddd` e o `number` diretamente, assim evitaria acessar esse índice zero mais de uma vez 🙂 

```suggestion
  const { ddd, number } = pathOr({}, ['phone_numbers', 0], register)
```

Assim basta usar `number` e `ddd` no corpo da condição logo abaixo

Reply:
@vagnervst Feito!